kiwi/volume_manager/btrfs.py

```diff
@@ -354,19 +357,48 @@ def _xml_pretty(self, toplevel_element):
         return xml_data_domtree.toprettyxml(indent="    ")
 
     def _create_quota_group_info(self):
-        snapper_configs = ''.join([self.mountpoint, '/@/etc/snapper/configs/'])
-        if os.path.exists(snapper_configs):
-            with open(snapper_configs + 'root', 'w') as snapper_root:
-                snapper_root.write(
-                    '{0}{1}'.format(self._get_quota_group_id(), os.linesep)
-                )
+        root_path = os.sep.join([self.mountpoint, '@/.snapshots/1/snapshot'])
+        snapper_default_conf = os.sep.join(
+            [root_path, 'etc/snapper/config-templates/default']
+        )
+        if os.path.exists(snapper_default_conf):
+            config_file = self._set_snapper_sysconfig_file(root_path)
+            if not os.path.exists(config_file):
+                shutil.copyfile(snapper_default_conf, config_file)
+            Command.run([
+                'chroot', root_path, 'snapper', '--no-dbus', 'set-config',
+                'QGROUP={0}'.format(self._get_quota_group_id())
+            ])
+
+    @classmethod
+    def _set_snapper_sysconfig_file(self, root_path):
```

Comment:
For this type of coding kiwi already provides the SysConfig class. See kiwi/utils/sysconfig.py
I suggest to make use of it and extend the class if needed, but so far I think it already provides what you need

Reply:
Oh, I missed that, checking it now